common/main/root.xml

```diff
@@ -67,13 +67,13 @@ for derived annotations.
 			<parseLenient sample="₹">[₨₹{Rp}{Rs}]</parseLenient>
 		</parseLenients>
 		<parseLenients scope="number" level="lenient">
-			<parseLenient sample="-">[\-‐‒–⁻₋−➖﹣－]</parseLenient>
-			<parseLenient sample=",">[,،٫、︐︑﹐﹑，､]</parseLenient>
+			<parseLenient sample="-">[\-ᱼ‐‒–⁻₋−➖﹣－]</parseLenient>
+			<parseLenient sample=",">[,،٫⹁、︐︑﹐﹑，､]</parseLenient>
 			<parseLenient sample="+">[+⁺₊➕﬩﹢＋]</parseLenient>
 		</parseLenients>
 		<parseLenients scope="number" level="stricter">
-			<parseLenient sample=",">[,٫︐﹐，]</parseLenient>
-			<parseLenient sample=".">[.․﹒．｡]</parseLenient>
+			<parseLenient sample=",">[,٫⹁︐﹐，]</parseLenient>
```

Comment:
This REVERSED COMMA looks ok (unlikely to ever happen) but it is a punctuation character at least.

Reply:
So we could take this but not the others, and make 2 changes in ICU. 

Or just make 3 changes in ICU.

Reply:
I don't understand what you mean by "just make 3 changes in ICU".  The failing ICU test is a data-driven test that ensures that parseLenients is a superset of the number formatting symbols in all locales.